src/main/resources/templates/sales/stats/sidebar.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head></head>
+
+<body>
+<!--/*@thymesVar id="key" type="java.lang.String"*/-->
+<nav class="col-sm-3 col-md-2 d-none d-sm-block sidebar" th:fragment="sidebar">
+    <ul class="nav nav-pills flex-column">
+        <li class="nav-item">
+            <a th:href="@{'/sales/stats/event/' + ${key}}" class="nav-link"
```

Comment:
replace `${key}` by `${event.getKey()}`, to make it clear which type of key it is.